djpyfs/djpyfs.py

```diff
@@ -20,6 +18,7 @@
 from django.conf import settings
 from fs.osfs import OSFS
 from fs_s3fs import S3FS
+from .models import FSExpirations
```

Comment:
Revert this change.

Reply:
This was giving pylint error in internal function. To clear quality tests, I had to move this to outer scope.